dist/rpm/singularity.spec.in

```diff
@@ -117,14 +126,26 @@ mkdir -p $RPM_BUILD_ROOT%{_mandir}/man1
 make DESTDIR=$RPM_BUILD_ROOT install man
 chmod 644 $RPM_BUILD_ROOT%{_sysconfdir}/singularity/actions/*
 
+%if "%{suse_version}" == "11"
+%clean
+/bin/rm -rf %{buildroot}
+%endif
+
 %files
 %attr(4755, root, root) %{_libexecdir}/singularity/bin/starter-suid
-%{_bindir}/*
+%{_bindir}/singularity
+%{_bindir}/run-singularity
```

Comment:
How is this an improvement to have to list more details in the %files section?

Reply:
Because .... rpmbuild on sles 11 was counting that as `/usr/bin/*` ... ... so, all of `/usr/bin` was being included... :/

Reply:
Oh how completely broken that is ... rpmbuild oh RHEL always gets a fresh buildroot.  Does that happen even with the explicit BuildRoot option also added by this PR?  That should give it its own independent build area.

Reply:
Unfortunately yes. At least while I was writing up this PR to work around it. :/